src/coreclr/vm/jithelpers.cpp

```diff
@@ -561,6 +561,46 @@ HCIMPL1_V(float, JIT_FloatRound, float val)
 }
 HCIMPLEND
 
+#ifndef TARGET_WINDOWS
+namespace
+{
+    bool isnan(float val)
```

Comment:
Nit: Moving this block around is unnecessary. It was more logical to have these methods right above the Rem helpers that are the only place where they are used in this file.

Reply:
*Added